packages/components/psammead-grid/README.md

```diff
@@ -0,0 +1,176 @@
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 2/8 columns</p>
+    </Grid>
+  </Grid>
+);
+```
+
+[Screenshot of this example without GEL Gutters and GEL Margins](./documentation/Screenshot\ example\ without\ gelMargins\ and\ gelGutters.png)
+
+####<a name="gutters-margins">Setting standard GEL gutters and GEL Margins</a>
+
+Using `enableGelGutters` and `enableGelMargins` on the wrapper Grid element.
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+	<Grid
+		wrapper
+		enableGelGutters
+    enableGelMargins
+		columns={{ group3: 6, group4: 8 }}
+	>
+    <Grid columns={{ group3: 6, group4: 6 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 6/8 columns</p>
+    </Grid>
+    <Grid columns={{ group3: 6, group4: 2 }}>
+      <p>Paragraph - for group 3 spans 6/6 columns, for group 4 spans 2/8 columns</p>
+    </Grid>
+  </Grid>
+);
+```
+
+####<a name="nested-grid">Nested grid example</a>
+
+Note that here, for each definition of a parent Grid, we need to use the `wrapper` prop. This sets the total number of columns via the `columns` prop. Then to define how many columns the child should span, you can use the `columns` prop on the child `Grid`.
+
+```jsx
+import Grid from '@bbc/psammead-grid';
+
+const MyComponent = () => (
+  <Grid
+    wrapper
+    columns={{ group2: 6, group3: 6 }}
+  >
+		<Grid
+		  wrapper
+      columns={{ group2: 6, group3: 6 }}
+    >
+      <Grid columns={{ group2: 6, group3: 3 }}>
+        <ExampleImage />
+      </Grid>
+      <Grid columns={{ group2: 6, group3: 3 }}>
+        <ExampleParagraph />
+      </Grid>
+    </Grid>
+    <Grid columns={{ group2: 2, group3: 2 }}>
+      <ExampleImage />
+    </Grid>
+    <Grid columns={{ group2: 4, group3: 4 }}>
+      <ExampleParagraph />
+    </Grid>
+  </Grid>
+);
+```
+
+[Screenshot of this example with GEL Gutters and GEL Margins enabled](./documentation/Screenshot\ example\ with\ gelMargins\ and\ gelGutters.png)
+
+** Setting GEL Max Widths**
+At the page level, you may want to have the GEL Grid's fixed max widths. 
+You can enable it with the prop `enableGelMaxWidths`. 
+
+For some pages, we will want to constrain the page to 1008px. You can use `enableGroupFourMaxWidth` for this. 
+
+### When to use this component
+
+This `Grid` component can be used for page-level layouts as well as for layouts of individual components. You can nest Grids within one another for more complex layouts. Please see the Storybook Stories for examples.
+
+### Frequenty Asked Questions
+
+- When should I use the `wrapper` prop?
+  - The `wrapper` prop should be used on a parent Grid element. It defines the grid, using the `columns` prop's values to define how many columns to define at each breakpoint. 
+  - It can and should be used whenever you're defining a new grid. [See nested grid example.](#nested-grid). It should be used in any case where you're defining a new nested grid.
```

Comment:
```suggestion
  - It can and should be used whenever you're defining a new grid. [See nested grid example](#nested-grid). It should be used in any case where you're defining a new nested grid.
```

Reply:
Updated the text entirely.